ezyrb/regular_grid.py

```diff
@@ -0,0 +1,131 @@
+"""
+Module for higher order interpolation on regular grids
+"""
+import numpy as np
+from scipy.interpolate import RegularGridInterpolator
+from .approximation import Approximation
+
+
+class RegularGrid(Approximation):
+    """
+    Multidimensional interpolator on regular grids.
+    See scipy's RegularGridInterpolator
+
```

Comment:
Please add the class documentation with the members description and possibly a minimal example. See [here](https://github.com/mathLab/EZyRB/blob/8a51f55e08d520e2c661d57c67a99e03dc1253ee/ezyrb/gpr.py#L15) for reference.